parser/src/main/java/com/microsoft/gctoolkit/parser/G1GCForwardReference.java

```diff
@@ -774,7 +774,7 @@ public void fullPhase(int integerGroup, String fullGCInternalPhase, double durat
     }
 
     public String toString() {
-        return gcType + ":" + getGCCause() + ":" + getDuration();
+        return getStartTime().toString() + " : " + gcType + " : " + getGCCause() + " : " + getDuration();
```

Comment:
Bikeshedding, but I prefer the order to be type, cause, datetimestamp, duration. My reasoning is that it keeps the two time elements together. The 'record' API that I've put into Aggregations follows that order.

Reply:
I'm following the convention in the GC logs of time, type, cause, duration